rasa/core/evaluation/markers.py

```diff
@@ -93,3 +93,20 @@ def _merge(cls, config_a: Dict, config_b: Dict) -> Dict:
             return copy_config_a
         else:
             return config_b
+
+    @classmethod
+    def _validate_config(cls, config: Dict, filename: Text = "") -> bool:
+        from jsonschema import validate
+        from jsonschema import ValidationError
+
+        try:
+            validate(config, MARKERS_SCHEMA)
```

Comment:
You could call the static method `config_format_spec()` instead of directly using the constant, or you could delete the static method if you're not using it anywhere else.

Reply:
Oh that was a mistake! I was calling the static method, but then moved stuff around while debugging my unit test. I don't call the static method anywhere else but I thought it might be useful to access the schema from the class itself? 🤔